internals/utils/src/index.ts

```diff
@@ -0,0 +1,35 @@
+export {default as ARG_COMMON} from './arg-common';
```

Comment:
theres no exact rational to the ordering here. just appended lines as i refactor files